src/ExperimentsServiceProvider.php

```diff
@@ -8,15 +8,27 @@ class ExperimentsServiceProvider extends ServiceProvider
 {
     public function boot()
     {
-        $this->publishConfig();
+        $this->publishes(
+            [
+                $this->configPath() => config_path('experiments.php'),
+            ],
+            'config'
+        );
+
+        if ($this->app->runningInConsole()) {
+            $this->commands(
+                [
+                    Console\InstallCommand::class,
+                ]
+            );
+        }
     }
 
     public function register()
     {
-        $this->commands(
-            [
-                Console\InstallCommand::class,
-            ]
+        $this->mergeConfigFrom(
```

Comment:
please don't add mergeConfigFrom

Reply:
why? (just asking so i know for any future PRs)

Reply:
there are some boilerplate example experiments in there. and these will actually be run if merged from the package.
```
'your-experiment-key' => [
        'control' => 'control-key',
        'test' => 'test-key',
    ],
```

Reply:
this does mean that new configs added to this package will be ignored by the downstream package if they have dumped the config

and i can just comment out the example experiment

Reply:
no leave please

Reply:
if I need a config added I will add mergeConfigFrom

Reply:
done